shared/login/reset/waiting.tsx

```diff
@@ -24,7 +22,23 @@ const Waiting = (props: Props) => {
     () => dispatch(nav.safeNavigateAppendPayload({path: ['login'], replace: true})),
     []
   )
-  const onSendAgain = todo
+
+  // TODO: visual feedback on click
+  const onSendAgain = React.useCallback(() => dispatch(AutoresetGen.createResetAccount({})), [])
+
+  React.useEffect(() => {
+    function tick() {
+      const newFormattedTime = Constants.formatTimeLeft(endTime)
+      if (formattedTime !== newFormattedTime) {
+        setFormattedTime(newFormattedTime)
+      }
+    }
+
+    const tickerID = addTicker(tick)
+    return function cleanup() {
```

Comment:
no need to name, `return () => removeTicker(tickerID)`

Reply:
this pattern is just so illegible to me without the name :/